server/cluster/coordinator_test.go

```diff
@@ -672,8 +672,11 @@ func (s *testCoordinatorSuite) TestPersistScheduler(c *C) {
 	_, err = schedule.CreateScheduler(schedulers.AdjacentRegionType, oc, storage, schedule.ConfigJSONDecoder([]byte("null")))
```

Comment:
I find another problem that when we use `ConfigJSONDecoder` here, the scheduler name will be empty and the path will only be `scheduler-config` instead of `scheduler-config/xxx`